spec/imsc-hrm.html

```diff
@@ -533,6 +540,7 @@ <h2>General</h2>
 
         <li>DUR<sub>I</sub>(E<sub>n</sub>) is the duration, in seconds, for painting the image subtitle content for <a>Intermediate
         Synchronic Document</a> E<sub>n</sub>, as specified in Section <a href="#paint-images">.</a>
+        <div class="issue" data-number="63"></div>
```

Comment:
Looks like this is probably the line that triggers the [validation failure](https://github.com/w3c/imsc-hrm/actions/runs/3738154309/jobs/6343983154#step:3:982), though the line numbers don't match up - I guess the validated line number is after Respec's processing has taken place.

Reply:
Yes, this line should move after closing of ul. div within li is prohibited.